upload/upload.go

```diff
@@ -71,6 +71,14 @@ func uploadFiles(files, outFiles []string, targetDir string, config *helpers.Con
 		return errors.New("no files to upload")
 	}
 
+	// Loop through the list of file paths and check if their names are valid
+	for _, filename := range outFiles {
+		err := helpers.CheckValidChars(filename)
```

Comment:
we should only have to do this check once regardless if `targetDir` is specified or not, but that means refactoring the entire upload function.
#save-for-later